packages/javascript/bh-shared-ui/src/views/Users/Users.tsx

```diff
@@ -59,24 +56,26 @@ const Users = () => {
     const [disable2FAError, setDisable2FAError] = useState('');
     const [disable2FASecret, setDisable2FASecret] = useState('');
 
-    const self = useAppSelector((state) => state.auth.user);
-    const hasSelectedSelf = useMemo(() => self?.id === selectedUserId!, [selectedUserId, self?.id]);
-
     const getSelfQuery = useQuery(['getSelf'], ({ signal }) =>
-        apiClient.getSelf({ signal }).then((res) => res.data.data)
+        apiClient.getSelf({ signal }).then((res) => res.data?.data)
+    );
+
+    const hasSelectedSelf = useMemo(
```

Comment:
This probably doesnt need to use a `useMemo()`, it looks like a pretty straightforward comparison! But did you notice performance was impacted without it?

Reply:
Tested it and doesn't seem to be a difference, removing!